dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -315,12 +316,7 @@ public Organization updateOrganizationDescription(@ApiParam(hidden = true) @Para
     @Operation(operationId = "getOrganizationMembers", summary = "Retrieve all members for an organization.", description = "Retrieve all members for an organization. Supports optional authentication.", security = @SecurityRequirement(name = JWT_SECURITY_DEFINITION_NAME))
     public Set<OrganizationUser> getOrganizationMembers(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user") @Auth Optional<User> user,
         @ApiParam(value = "Organization ID.", required = true) @Parameter(description = "Organization ID.", name = "organizationId", in = ParameterIn.PATH, required = true) @PathParam("organizationId") Long id) {
-        Set<OrganizationUser> acceptedUsers = getOrganizationByIdOptionalAuth(user, id).getUsers()
-            .stream()
-            .filter(orgUser -> orgUser.isAccepted())
-            .collect(Collectors.toSet());
-
-        return acceptedUsers;
+        return getOrganizationByIdOptionalAuth(user, id).getUsers();
```

Comment:
I didn't try running this, so I might be wrong, but won't this allow all Dockstore users to view all invited/rejected users of an org?

If so, I don't think that should be allowed; only org admins should have access to invites and rejections.